src/templates/components/CapiMultipleCard.svelte

```diff
@@ -71,6 +71,13 @@
 		overflow: hidden;
 		/** This is needed to absolutely position the hover overlay */
 		position: relative;
+
+		@media (max-width: 739.9px) {
```

Comment:
very precise this isn't it. Can we make use of `@media (max-width < 740px) {` or is that too modern?